app/models/miq_worker/container_common.rb

```diff
@@ -77,8 +77,8 @@ def resource_constraints
       mem_request   = self.class.worker_settings[:memory_request]
       cpu_request   = self.class.worker_settings[:cpu_request_percent]
 
-      raise ArgumentError, "cpu_request_percent cannot exceed cpu_threshold_percent" if (cpu_request || 0) > (cpu_limit || Float::INFINITY)
-      raise ArgumentError, "memory_request cannot exceed memory_threshold"           if (mem_request || 0) > (mem_limit || Float::INFINITY.megabytes)
+      raise ArgumentError, "#{self.class.name} - cpu_request_percent (#{cpu_request}) cannot exceed cpu_threshold_percent (#{cpu_limit})" if (cpu_request || 0) > (cpu_limit || Float::INFINITY)
+      raise ArgumentError, "#{self.class.name} - memory_request (#{mem_request}) cannot exceed memory_threshold (#{mem_limit})"           if (mem_request || 0) > (mem_limit || Float::INFINITY.megabytes)
```

Comment:
should `#{mem_request}` be `#{mem_request.inspect}` to print `nil` when there is no value?  Same would apply to the others

Reply:
Yes...that would be good.

Reply:
Or it should do the `|| 0` as in the conditional.

Reply:
hopefully it will not fail when either is nil, so leaning against `|| 0` as the `nil` may help us in diagnostics.  Thoughts?

Reply:
Sure, `#{mem_request.inspect}` and similar changes sound good. It will be more obvious it's nil and not an empty string.